edit-post/components/header/more-menu/style.scss

```diff
@@ -24,4 +24,10 @@
 	.components-menu-items__group:not(:last-child) {
   		border-bottom: 1px solid $light-gray-500;
 	}
+
+	// Ensure that even SVG icons that don't include the .dashicon class are colored
+	.components-icon-button svg {
```

Comment:
Is this something we want to make specific to the icon buttons used within the more menu? Or should it be applied to `<IconButton>` generally?

Reply:
This should be applied to icon buttons generally, the way I see it (always a chance I'm missing something). The idea being that any button with an icon should have consistent coloring that is the same as the text. 

The alternative is that plugins can add a multi colored SVG icon for _their_ particular plugin button, to "stand out", by simply not including the `dashicon` class for an SVG they pass as an inline element. This would be bad for consistency.

I mean a plugin can always do this regardless by using `!important` or whatnot, but no reason to make it easy.